packages/island/src/theme-default/layout/DocLayout/index.tsx

```diff
@@ -12,11 +12,18 @@ export interface DocLayoutProps {
   afterDoc?: React.ReactNode;
   beforeOutline?: React.ReactNode;
   afterOutline?: React.ReactNode;
+  isSidebarOpen?: boolean;
 }
 
 export function DocLayout(props: DocLayoutProps) {
-  const { beforeDocFooter, beforeDoc, afterDoc, beforeOutline, afterOutline } =
-    props;
+  const {
+    beforeDocFooter,
+    beforeDoc,
+    afterDoc,
+    beforeOutline,
+    afterOutline,
+    isSidebarOpen
```

Comment:
I don't it's a good way to put `isSidebarOpen` as a prop.It's a little messy and not clear enough. However, `useSidebar` should can be access the state directly and make the code clean.